### PR TITLE
Clarified ambiguous wording

### PR DESCRIPTION
**Pull request by ThomasLandauer**
In this context, "even" can be misunderstood as "actually".

### CODE REVIEW

File: components/dom_crawler.rst [hunk 1]
```diff
@@ -87,7 +87,7 @@ Anonymous function can be used to filter with more complex criteria::
     $crawler = $crawler
         ->filter('body > p')
         ->reduce(function (Crawler $node, $i) {
-            // filter even nodes
+            // filter every second node
```
Comment (xabbuh) [inline components/dom_crawler.rst]: What about "filter every other node"?

**Comment by javiereguiluz** [comment]
:+1: @ThomasLandauer thanks for this small but nice improvement!

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks Thomas! I've merged this into the 2.3 branch. Cheers!

**Pull request closed without merging**